packages/@okta/vuepress-site/docs/reference/csi-delauth-hook/index.md

```diff
@@ -0,0 +1,541 @@
+---
+title: Custom Source Integration Delegated Authentication Inline Hook Reference
+excerpt: Add custom logic to the Custom Source Integration Delegated Authentication process.
+---
+
+# Custom Source Integration Delegated Authentication Inline Hook Reference
+
+<ApiLifecycle access="beta" />
+
+This page provides reference documentation for:
+
+- JSON objects contained in the outbound request from Okta to your external service
+
+- JSON objects you can include in your response
+
+This information is specific to the Custom Source Integration Delegated Authentication Inline Hook, one type of Inline Hook supported by Okta.
+
+## See Also
+
+For a general introduction to Okta Inline Hooks, see [Inline Hooks](/docs/concepts/inline-hooks/).
+
+For information on the API for registering external service endpoints with Okta, see [Inline Hooks Management API](/docs/reference/api/inline-hooks/).
+
+## About
+
+The Custom Source Integration Delegated Authentication Inline Hook enables you to delegate authentication of a user to an external identity source.
+
+The hook supports a few operations. It first allows you to import a user into Okta after a successful authentication. From there it allows delegated authentication of the user to the external source, and fetches the latest profile of the user from the external source.
```

Comment:
Do we want to call out that this scenario is commonly known as "Just In Time (JIT) provisioning?"

Reply:
It's one of Trevor's recommendations in inline hook data contract review to not use JIT term in the inline hook level as the hook remains a generic channel, so I have been intentionally avoid using that term as much as possible in inline hook doc and code.